diff -urN bluefish-0.7/Makefile.in bluefish-0.7.new/Makefile.in
--- bluefish-0.7/Makefile.in	Tue Jul 24 15:03:38 2001
+++ bluefish-0.7.new/Makefile.in	Sat Nov 24 16:24:20 2001
@@ -70,8 +70,8 @@
 	@cd po; $(MAKE) install
 	@cd man; $(MAKE) install
 	@cd data; $(MAKE) install
-	$(INSTALL) -d -m 755 $(pixmapsdir)
-	$(INSTALL) -m 644 ./pixmaps/bluefish_icon.xpm $(pixmapsdir)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(pixmapsdir)
+	$(INSTALL) -m 644 ./pixmaps/bluefish_icon.xpm $(DESTDIR)$(pixmapsdir)
 	@cd ${srcdir}/ && $(MAKE) install
 
 
diff -urN bluefish-0.7/data/Makefile.in bluefish-0.7.new/data/Makefile.in
--- bluefish-0.7/data/Makefile.in	Tue Jul 24 15:03:39 2001
+++ bluefish-0.7.new/data/Makefile.in	Sat Nov 24 16:23:21 2001
@@ -79,8 +79,8 @@
 distclean: patchclean
 
 install: all
-	$(INSTALL) -d -m 755 $(pkgdatadir)/man1
-	$(INSTALL) -m 644 ./$(OBJS) $(pkgdatadir)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(pkgdatadir)/man1
+	$(INSTALL) -m 644 ./$(OBJS) $(DESTDIR)$(pkgdatadir)
 
 uninstall:
 	/bin/rm -f $(pkgdatadir)/$(OBJS)
diff -urN bluefish-0.7/man/Makefile.in bluefish-0.7.new/man/Makefile.in
--- bluefish-0.7/man/Makefile.in	Tue Jul 24 15:03:40 2001
+++ bluefish-0.7.new/man/Makefile.in	Sat Nov 24 16:23:00 2001
@@ -53,8 +53,8 @@
 distclean: patchclean
 
 install: all
-	$(INSTALL) -d -m 755 $(mandir)/man1
-	$(INSTALL) -m 644 ./$(OBJS) $(mandir)/man1/$(OBJS)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(mandir)/man1
+	$(INSTALL) -m 644 ./$(OBJS) $(DESTDIR)$(mandir)/man1/$(OBJS)
 
 uninstall:
 	/bin/rm -f $(mandir)/$(OBJS)
diff -urN bluefish-0.7/src/Makefile.in bluefish-0.7.new/src/Makefile.in
--- bluefish-0.7/src/Makefile.in	Tue Jul 24 15:03:46 2001
+++ bluefish-0.7.new/src/Makefile.in	Sat Nov 24 16:23:11 2001
@@ -77,8 +77,8 @@
 	rm -f config.h
 
 install: all
-	$(INSTALL_DIR) $(bindir)
-	$(INSTALL_PROGRAM) ./bluefish $(bindir)/bluefish
+	$(INSTALL_DIR) $(DESTDIR)$(bindir)
+	$(INSTALL_PROGRAM) ./bluefish $(DESTDIR)$(bindir)/bluefish
 
 uninstall:
 	/bin/rm -f $(bindir)/bluefish
