--- bluefish-2.2.0/configure.ac~	2011-11-06 03:11:55.000000000 +0100
+++ bluefish-2.2.0/configure.ac	2011-12-22 12:43:59.047287815 +0100
@@ -34,7 +34,7 @@
 dnl ************************
 dnl   I18N options
 dnl ************************
-m4_define([_BF_LINGUAS], [ar bg ca cs da de el en es eu fa fi fr gl hu it ja ko nb nl nn pl pt pt_BR ro ru sk sr sv ta tr uk zh_CN zh_TW])
+m4_define([_BF_LINGUAS], [ar bg ca cs da de el en es eu fa fi fr gl hu it ja ko nb nl nn pl pt pt_BR ro ru sk sr@Latin sv ta tr uk zh_CN zh_TW])
 #ALL_LINGUAS="bg cs da de el es eu fi fr gl hu it ja ko nb nl pl pt pt_BR ro ru sk sr sv ta tr uk zh_CN zh_TW"
 AC_SUBST([ALL_LINGUAS], "_BF_LINGUAS")
 GETTEXT_PACKAGE=AC_PACKAGE_NAME
