--- Makefile.in~	Tue Jul  6 08:57:38 1999
+++ Makefile.in	Sun Jul 11 20:57:09 1999
@@ -45,10 +45,10 @@
 	rm -f *.orig *.rej
 
 install:
-	$(INSTALL) -d -m 755 @install_location@
-	$(INSTALL) -m 644 ./data/php3_functions @install_location@/php3_functions
-	$(INSTALL) -m 644 ./data/ssi_functions @install_location@/ssi_functions
-	$(INSTALL) -m 644 ./data/rxml_functions @install_location@/rxml_functions	
+	$(INSTALL) -d -m 755 $(DESTDIR)@install_location@
+	$(INSTALL) -m 644 ./data/php3_functions $(DESTDIR)@install_location@/php3_functions
+	$(INSTALL) -m 644 ./data/ssi_functions $(DESTDIR)@install_location@/ssi_functions
+	$(INSTALL) -m 644 ./data/rxml_functions $(DESTDIR)@install_location@/rxml_functions	
 	@cd ${srcdir}/ && $(MAKE) install
 
 uninstall:
--- src/Makefile.in~	Tue Jul  6 08:57:44 1999
+++ src/Makefile.in	Sun Jul 11 20:58:08 1999
@@ -53,8 +53,8 @@
 	rm -f config.h
 
 install: all
-	$(INSTALL) -d -m 755 $(bindir)
-	$(INSTALL) -s -m 755 ./bluefish $(bindir)/bluefish
+	$(INSTALL) -d -m 755 $(DESTDIR)$(bindir)
+	$(INSTALL) -s -m 755 ./bluefish $(DESTDIR)$(bindir)/bluefish
 
 uninstall:
 	/bin/rm -f $(bindir)/bluefish
