diff -ruN bluefish-0.9.orig/Makefile.in bluefish-0.9/Makefile.in
--- bluefish-0.9.orig/Makefile.in	Wed Feb 19 23:39:25 2003
+++ bluefish-0.9/Makefile.in	Wed Feb 19 23:45:25 2003
@@ -29,7 +29,7 @@
 exec_prefix=@exec_prefix@
 bindir=@bindir@
 datadir=@datadir@
-pkgdatadir=$(datadir)/$(PACKAGE)
+pkgdatadir=$(DESTDIR)/$(datadir)/$(PACKAGE)
 pixmapsdir=@prefix@/share/pixmaps
 
 INSTALL = @INSTALL@
@@ -64,12 +64,12 @@
 	rm -f *.orig *.rej
 
 install: src/bluefish
-	pkgdatadir=@pkgdatadir@
+	pkgdatadir=$(DESTDIR)/@pkgdatadir@
 	$(INSTALL_DIR) $(pkgdatadir)
 	$(INSTALL_DATA) images/bluefish_splash.png $(pkgdatadir)
 	@cd icons/ && $(MAKE) install
 	if [ -d $(iconpath) ]; then \
-		$(INSTALL_DATA) inline_images/bluefish_icon1.png $(iconpath)/bluefish-icon.png ;\
+		# $(INSTALL_DATA) inline_images/bluefish_icon1.png $(iconpath)/bluefish-icon.png ;\
 	fi
	@cd src/ && $(MAKE) install
 	@cd po/ && $(MAKE) install
diff -ruN bluefish-0.9.orig/data/Makefile.in bluefish-0.9/data/Makefile.in
--- bluefish-0.9.orig/data/Makefile.in	Wed Feb 19 23:39:25 2003
+++ bluefish-0.9/data/Makefile.in	Wed Feb 19 23:43:55 2003
@@ -12,7 +12,7 @@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
 bindir = @bindir@
-pkgdatadir = @pkgdatadir@
+pkgdatadir = $(DESTDIR)/@pkgdatadir@
 datadir = @datadir@
 localedir = $(datadir)/locale
 
@@ -33,11 +33,11 @@
 		$(INSTALL_DATA) $$file $(pkgdatadir)/$$file ;\
 	done
 	if [ -d $(gnome2menupath) ] ; then \
-		$(INSTALL_DATA) bluefish.desktop $(gnome2menupath) ;\
+		$(INSTALL_DATA) bluefish.desktop $(DESTDIR)/$(gnome2menupath) ;\
 	fi
 	if [ -d $(gnome1menupath) ] ; then \
-		$(INSTALL_DIR) $(gnome1menupath)/Applications ;\
+		$(INSTALL_DIR) $(DESTDIR)/$(gnome1menupath)/Applications ;\
-		$(INSTALL_DATA) bluefish.desktop $(gnome1menupath)/Applications ;\
+		$(INSTALL_DATA) bluefish.desktop $(DESTDIR)/$(gnome1menupath)/Applications ;\
 	fi
 
 uninstall:
diff -ruN bluefish-0.9.orig/icons/Makefile.in bluefish-0.9/icons/Makefile.in
--- bluefish-0.9.orig/icons/Makefile.in	Wed Feb 19 23:39:25 2003
+++ bluefish-0.9/icons/Makefile.in	Wed Feb 19 23:42:52 2003
@@ -22,7 +22,7 @@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
 bindir = @bindir@
-pkgdatadir = @pkgdatadir@
+pkgdatadir = $(DESTDIR)/@pkgdatadir@
 
 infodir = $(prefix)/info
 libdir = $(prefix)/lib/gnudl
diff -ruN bluefish-0.9.orig/po/Makefile.in bluefish-0.9/po/Makefile.in
--- bluefish-0.9.orig/po/Makefile.in	Wed Feb 19 23:39:24 2003
+++ bluefish-0.9/po/Makefile.in	Wed Feb 19 23:45:52 2003
@@ -14,7 +14,7 @@
 bindir = @bindir@
 pkgdatadir = @pkgdatadir@
 datadir = @datadir@
-localedir = $(datadir)/locale
+localedir = $(DESTDIR)/$(datadir)/locale
 
 INSTALL = @INSTALL@
 INSTALL_DIR = $(INSTALL) -d -m 755 
diff -ruN bluefish-0.9.orig/src/Makefile.in bluefish-0.9/src/Makefile.in
--- bluefish-0.9.orig/src/Makefile.in	Wed Feb 19 23:39:25 2003
+++ bluefish-0.9/src/Makefile.in	Wed Feb 19 23:44:37 2003
@@ -21,7 +21,7 @@
 srcdir = @srcdir@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
-bindir = @bindir@
+bindir = $(DESTDIR)/@bindir@
 pkgdatadir = @pkgdatadir@
 
 infodir = $(prefix)/info
--- bluefish-0.9/Makefile.in~	Thu Feb 20 00:29:51 2003
+++ bluefish-0.9/Makefile.in	Thu Feb 20 00:32:12 2003
@@ -68,9 +68,9 @@
 	$(INSTALL_DIR) $(pkgdatadir)
 	$(INSTALL_DATA) images/bluefish_splash.png $(pkgdatadir)
 	@cd icons/ && $(MAKE) install
-	if [ -d $(iconpath) ]; then \
+#	if [ -d $(iconpath) ]; then \
 		# $(INSTALL_DATA) inline_images/bluefish_icon1.png $(iconpath)/bluefish-icon.png ;\
-	fi
+#	fi
 	@cd src/ && $(MAKE) install
 	@cd po/ && $(MAKE) install
 	@cd data/ && $(MAKE) install
