--- bluefish-0.13/data/bluefish.desktop.in.orig	2004-04-04 22:13:52.000000000 +0200
+++ bluefish-0.13/data/bluefish.desktop.in	2004-04-09 21:03:56.000000000 +0200
@@ -1,8 +1,9 @@
 [Desktop Entry]
 Name=Bluefish Editor
 Comment=Web Development Editor
+Comment[pl]=Edytor do tworzenia stron internetowych
 Exec=bluefish -n
-Icon=ICONDIR/bluefish-icon.png
+Icon=bluefish.png
 Terminal=false
 Type=Application
 Categories=Application;Development;
