--- bluefish-0.13/configure.ac~	Mon Jul 26 00:30:34 2004
+++ bluefish-0.13/configure.ac	Mon Jul 26 00:34:25 2004
@@ -30,7 +30,7 @@
 AC_DEFINE_UNQUOTED(CONFIGURE_OPTIONS, "$0 $*")
 
 # these are the languages that will be installed, so these should be finished
-ALL_LINGUAS="da de es fi fr it no pl pt pt_BR ru sr sv zh_CN"
+ALL_LINGUAS="bg cs da de el es fi fr hu it ja nb nl pl pt pt_BR ro ru sr@Latn sv ta zh_CN"
 AC_SUBST(ALL_LINGUAS)
   
 dnl ************************   
--- bluefish-0.13/configure~	Mon Jul 26 00:34:25 2004
+++ bluefish-0.13/configure	Mon Jul 26 00:34:25 2004
@@ -1378,7 +1378,7 @@
 
 
 # these are the languages that will be installed, so these should be finished
-ALL_LINGUAS="da de es fi fr it no pl pt pt_BR ru sr sv zh_CN"
+ALL_LINGUAS="bg cs da de el es fi fr hu it ja nb nl pl pt pt_BR ro ru sr@Latn sv ta zh_CN"
 
 
 
--- bluefish-0.13/po/Makefile.in~	Mon Jul 26 00:51:30 2004
+++ bluefish-0.13/po/Makefile.in	Mon Jul 26 00:51:30 2004
@@ -24,7 +24,7 @@
 XGETTEXT = @XGETTEXT@
 MSGMERGE = @MSGMERGE@
 
-SRCS = bg.po cs.po da.po de.po el.po es.po fi.po fr.po hu.po it.po ja.po nl.po no.po pl.po pt.po pt_BR.po ro.po ru.po sr.po sv.po ta.po zh_CN.po
+SRCS = bg.po cs.po da.po de.po el.po es.po fi.po fr.po hu.po it.po ja.po nb.po nl.po pl.po pt.po pt_BR.po ro.po ru.po sr@Latn.po sv.po ta.po zh_CN.po
 .SUFFIXES: .po .gmo
 OBJS = $(SRCS:.po=.gmo)
 
