--- Makefile.in~	Sat Jun 26 14:28:42 1999
+++ Makefile.in	Mon Jun 28 21:54:50 1999
@@ -45,8 +45,8 @@
 	rm -f *.orig *.rej
 
 install:
-	$(INSTALL) -d -m 755 @install_location@
-	$(INSTALL) -m 755 ./bluefish.inst @install_location@/bluefish.inst
+	$(INSTALL) -d -m 755 $(DESTDIR)@install_location@
+	$(INSTALL) -m 755 ./bluefish.inst $(DESTDIR)@install_location@/bluefish.inst
 	@cd ${srcdir}/ && $(MAKE) install
 
 uninstall:
--- src/Makefile.in~	Sat Jun 26 14:28:52 1999
+++ src/Makefile.in	Mon Jun 28 21:44:47 1999
@@ -53,7 +53,7 @@
 	rm -f config.h
 
 install: all
-	$(INSTALL) -s -m 755 ./bluefish $(bindir)/bluefish
+	$(INSTALL) -s -m 755 ./bluefish $(DESTDIR)$(bindir)/bluefish
 
 uninstall:
 	/bin/rm -f $(bindir)/bluefish
