diff -Nru bluefish-0.4/Makefile.in bluefish-0.4.new/Makefile.in
--- bluefish-0.4/Makefile.in	Mon Apr 17 23:31:14 2000
+++ bluefish-0.4.new/Makefile.in	Mon May 15 00:47:52 2000
@@ -62,13 +62,13 @@
 install:
 	@$(MAKE) -C po install
 	@$(MAKE) -C man install
-	$(INSTALL) -d -m 755 $(datadir)
-	$(INSTALL) -d -m 755 $(pkgdatadir)
-	$(INSTALL) -d -m 755 $(pixmapsdir)
-	$(INSTALL) -m 644 ./pixmaps/bluefish_icon.xpm $(pixmapsdir)
-	$(INSTALL) -m 644 ./data/php3_functions $(pkgdatadir)/php3_functions
-	$(INSTALL) -m 644 ./data/ssi_functions $(pkgdatadir)/ssi_functions
-	$(INSTALL) -m 644 ./data/rxml_functions $(pkgdatadir)/rxml_functions	
+	$(INSTALL) -d -m 755 $(DESTDIR)$(datadir)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(pkgdatadir)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(pixmapsdir)
+	$(INSTALL) -m 644 ./pixmaps/bluefish_icon.xpm $(DESTDIR)$(pixmapsdir)
+	$(INSTALL) -m 644 ./data/php3_functions $(DESTDIR)$(pkgdatadir)/php3_functions
+	$(INSTALL) -m 644 ./data/ssi_functions $(DESTDIR)$(pkgdatadir)/ssi_functions
+	$(INSTALL) -m 644 ./data/rxml_functions $(DESTDIR)$(pkgdatadir)/rxml_functions	
 	@cd ${srcdir}/ && $(MAKE) install
 
 
diff -Nru bluefish-0.4/man/Makefile.in bluefish-0.4.new/man/Makefile.in
--- bluefish-0.4/man/Makefile.in	Sat Mar  4 16:07:23 2000
+++ bluefish-0.4.new/man/Makefile.in	Mon May 15 00:48:15 2000
@@ -40,11 +40,8 @@
 INSTALL = @INSTALL@
 
 SRCS = bluefish.1
-OBJS = bluefish.1.gz
 
-all: bluefish
-bluefish:
-	gzip -c $(SRCS) > $(OBJS)
+all:
 
 clean:
 	rm -f $(OBJS) core *~ *.bak
@@ -55,5 +52,5 @@
 distclean: patchclean
 
 install: all
-	$(INSTALL) -d -m 755 $(mandir)
-	$(INSTALL) -m 644 ./$(OBJS) $(mandir)/$(OBJS)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(mandir)
+	$(INSTALL) -m 644 ./$(SRCS) $(DESTDIR)$(mandir)/$(OBJS)
diff -Nru bluefish-0.4/src/Makefile.in bluefish-0.4.new/src/Makefile.in
--- bluefish-0.4/src/Makefile.in	Mon Apr 17 23:31:16 2000
+++ bluefish-0.4.new/src/Makefile.in	Mon May 15 00:45:58 2000
@@ -68,8 +68,8 @@
 	rm -f config.h
 
 install: all
-	$(INSTALL) -d -m 755 $(bindir)
-	$(INSTALL) -s -m 755 ./bluefish $(bindir)/bluefish
+	$(INSTALL) -d -m 755 $(DESTDIR)$(bindir)
+	$(INSTALL) -s -m 755 ./bluefish $(DESTDIR)$(bindir)/bluefish
 
 uninstall:
 	/bin/rm -f $(bindir)/bluefish
